### ISSUE TITLE
pooling, dropout and flatten show layer.trainable as True

### ISSUE THREAD

**Issue by tadbeer** [keras-team/keras#10328]
I am using VGG19 with imagenet weights to initialize my model (a simple classifier) and while freezing certain layers, I saw something that doesn't make sense. We know that there are no trainable parameters in layers like max_pooling2D, Dropout as well as Flatten, but when i use:
for layer in model.layers:
    print(layer.trainable)
it shows True for these layers. At first I thought it was maybe set as default for all the layers, but for Input layer, it shows false. Kindly let me know if there are parameters apart from their own that need to be trained, if any.
